Tasks/AzureRmWebAppDeploymentV5/_buildConfigs/Node20/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "vsts-tasks-azurermappservicedeployment",
+  "version": "1.0.0",
+  "description": "Azure Pipelines Azure RM App Service Deployment",
+  "main": "AzureRmWebAppDeployment.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/Microsoft/azure-pipelines-tasks.git"
+  },
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Microsoft/azure-pipelines-tasks/issues"
+  },
+  "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
+  "dependencies": {
+    "@types/mocha": "^5.2.7",
+    "@types/node": "^20.3.1",
+    "@types/q": "1.0.7",
+    "agent-base": "^6.0.2",
+    "azure-pipelines-task-lib": "4.11.0",
+    "azure-pipelines-tasks-azure-arm-rest": "^3.246.4",
```

Comment:
Hi @ShilpiRachna1 
Please update here the package version of "azure-pipelines-tasks-azure-arm-rest".